apps/postgresql-server/Dockerfile

```diff
@@ -10,24 +10,44 @@ RUN \
         /opt/postgresql-server/bin/ \
         /opt/postgresql-server/schema/ \
     && \
+    apt-get -y --no-install-recommends install python3 python3-pip python3-setuptools && \
+    #
+    # Upgrade Pip
+    pip3 install -U pip && \
+    #
+    # https://github.com/pypa/pip/issues/5221#issuecomment-382069604
+    hash -r pip3 && \
+    apt-get -y --no-install-recommends install build-essential python3-dev libffi-dev libpq-dev && \
+    #
+    # Install package to manage schema migrations
+    pip3 install yandex-pgmigrate==1.0.6 && \
+    #
+    # Remove temporary dependencies
+    apt-get -y remove build-essential python3-dev libffi-dev libpq-dev && \
+    #
+    # Cleanup
+    apt-get -y autoremove && \
+    rm -rf /root/.cache/ && \
+    #
     true
 
 # Copy helper scripts, schema, migrations
 COPY bin/* /opt/postgresql-server/bin/
 COPY schema/ /opt/postgresql-server/schema/
 
-# Initialize data volume, create users, a database, and initialize it with
-# schema
+# Copy helper scripts, schema, migrations, pgmigrate callbacks/config
+RUN mkdir -p /opt/mediacloud/
+COPY bin/ /opt/mediacloud/bin/
+COPY pgmigrate-callbacks/ /opt/mediacloud/pgmigrate-callbacks/
+COPY migrations/ /opt/mediacloud/migrations/
+COPY migrations.yml /opt/mediacloud/migrations.yml
+RUN cd /opt/mediacloud
```

Comment:
I don't think you need to `cd` anywhere here at all, but in any case it [doesn't do what it you intended for it to do](https://stackoverflow.com/a/58848691) - you'd need `WORKDIR` for that.